### PR TITLE
adding-tables-connect-sequelize

### PR DESCRIPTION
**Pull request by cr7hardwork**

### CODE REVIEW

File: package.json
Comment (vardansafaryan0407) [inline package.json:1]: @cr7hardwork please revert package.json and package-lock.json files

File: src/app.module.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ import {CoreModule} from "./core/core.module";
 @Module({
     imports: [
         SequelizeModule.forRoot(CONFIG_DB as SequelizeOptions),
+        SequelizeModule.forRoot(CONFIG_DB_SI as SequelizeOptions),
```
Comment (vardansafaryan0407) [inline src/app.module.ts:16]: @cr7hardwork remove additinal configs

File: src/core/models/countries.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { AutoIncrement, Column, Model, PrimaryKey, Table } from "sequelize-typescript";
+
```
Comment (vardansafaryan0407) [inline src/core/models/countries.ts:2]: @cr7hardwork rename properties according our discussion

File: src/core/models/equity.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        type: 'INT', 
+        allowNull: true 
+    })
+    actual_percent: number; 
+
+    @Column({
+        type: 'INT', 
+        allowNull: false
+    })
+    project_members_id: number;
```
Comment (vardansafaryan0407) [inline src/core/models/equity.ts:33]: @cr7hardwork please add foireign key for this field

File: src/countries/countries.service.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+@Injectable()
+export class CountriesService  extends BaseService<Countries>{
+
+    constructor(repository : CountriesRepository){
+        super(repository)
+    }
+
+
+
+    async getAll() : Promise<Countries[]>{
```
Comment (vardansafaryan0407) [inline src/countries/countries.service.ts:15]: @cr7hardwork please reuse base service method for this, and please apply that for all methods

File: package-lock.json
Comment (vardansafaryan0407) [inline package-lock.json:1]: @cr7hardwork revert package-lock json file

File: src/app.module.ts [hunk 5]
```diff
@@ -2,7 +2,7 @@ import {Module} from '@nestjs/common';
 import {AppController} from './app.controller';
 import {AppService} from './app.service';
 import {AuthModule} from './auth/auth.module';
-import {UserModule} from './user/user.module';
```
Comment (vardansafaryan0407) [inline src/app.module.ts:5]: @cr7hardwork why did you change existing module to the old one?

File: src/auth/auth.module.ts [hunk 6]
```diff
@@ -3,12 +3,11 @@ import {AuthController} from "./auth.controller";
 import {AuthService} from './services/auth.service';
 import {AuthRepository} from './repositories/auth.repository';
 import {JwtSharedModule} from "../core/modules/jwt.module";
-import {UserModule} from "../user/user.module";
```
Comment (vardansafaryan0407) [inline src/auth/auth.module.ts:6]: @cr7hardwork same here

File: src/auth/services/auth.service.ts [hunk 7]
```diff
@@ -3,15 +3,15 @@ import {AuthRepository} from "../repositories/auth.repository";
 import {BaseService} from "../../core/services/base.service";
 import {UserSignInDto} from "../dto/user-sign-in.dto";
 import {JwtService} from "@nestjs/jwt";
-import {UserService} from "../../user/user.service";
```
Comment (vardansafaryan0407) [inline src/auth/services/auth.service.ts:6]: @cr7hardwork this is wrong, this is wring folder generally

File: src/auth/repositories/auth.repository.ts [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import {Injectable} from '@nestjs/common';
 import {BaseRepository} from "../../core/repositories/base.repository";
-import {User} from "../../user/user";
```
Comment (vardansafaryan0407) [inline src/auth/repositories/auth.repository.ts:3]: @cr7hardwork why did you cahneg paths to absolut?

File: src/auth/services/auth.service.ts [hunk 9]
```diff
@@ -3,15 +3,15 @@ import {AuthRepository} from "../repositories/auth.repository";
...
+import { UsersService } from 'src/core-database/users/users.service';
 import {PasswordUtils} from "../../utils/password.utils";
-import {User} from "../../user/user";
```
Comment (vardansafaryan0407) [inline src/auth/services/auth.service.ts:8]: same here

File: src/config/config.db.ts [hunk 10]
```diff
@@ -2,9 +2,9 @@ export const CONFIG_DB = {
     dialect: 'mysql',
     host: 'localhost',
     port: 3306,
-    username: 'root',
```
Comment (vardansafaryan0407) [inline src/config/config.db.ts:5]: @cr7hardwork this won't work

File: src/user/user.controller.ts
Comment (vardansafaryan0407) [inline src/user/user.controller.ts:1]: @cr7hardwork why did you delete user module and created it in a wrong folder?

Comment (cr7hardwork) [reply src/user/user.controller.ts:1]: i have a user in my core-database folder, and all logic i think is correct please check

Comment (vardansafaryan0407) [reply src/user/user.controller.ts:1]: 1. It's wrong to put all these models ain one folder
2. folder name is wrong, please check the structure I had earlier and fix everything according that

File: package.json
Comment (vardansafaryan0407) [reply package.json:1]: @cr7hardwork this still is not reverted

**Review by vardansafaryan0407 — APPROVED** [review]
